sdk/azcore/doc.go

```diff
@@ -5,160 +5,125 @@
 // license that can be found in the LICENSE file.
 
 /*
-Package azcore implements an HTTP request/response middleware pipeline whose
-policy objects mutate an HTTP request's URL, query parameters, and/or headers before
-the request is sent over the wire.
+Package azcore implements an HTTP request/response middleware pipeline.
 
-Not all policy objects mutate an HTTP request; some policy objects simply impact the
-flow of requests/responses by performing operations such as logging, retry policies,
-timeouts, failure injection, and deserialization of response payloads.
+The middleware consists of three components.
+- One or more Policy objects.
```

Comment:
Somewhat minor issue, I don't like calling these "objects" as it makes me think of OOP.  Open to suggestions.

Reply:
Instance(s)?

Reply:
That could work.  Also thought of "implementation(s)".

Reply:
Changed to instance, I like it.